src/_zkapauthorizer/model.py

```diff
@@ -129,7 +116,7 @@ def initialize_database(conn: Connection) -> None:
         # be enabled on a per-connection basis.  This is a helpful feature to
         # ensure consistency so we want it enforced and we use it in our
         # schema.
-        cursor.execute("PRAGMA foreign_keys = ON")
+        cursor.execute("PRAGMA foreign_keys = ON", ())
```

Comment:
mypy doesn't have variadic functions so we have to pass an empty argument tuple to execute now.